### ISSUE TITLE
Radial menus not great at following their anchor

### ISSUE THREAD

**Issue by I-VAPE-VOX-CLOACA-EVERY-DAY-OF-MY-LIFE** [vgstation-coders/vgstation13#19825]
If the anchor is something on the game world (e.g. not the user itself) then the menu doesn't really make any attempt to follow it
![dreamseeker_2018-09-27_05-47-03](https://user-images.githubusercontent.com/40967382/46134410-2ecda480-c219-11e8-80db-b4231579785a.png)

They also don't inherit their anchor's pixel_x or pixel_y

**Comment by DeityLink** [comment]
The easier way to fix the pixel offset is to anchor to the turf instead of the object, though that only works if the object is anchored itself.
In the above case where you can just pull the object, I'd probably use an input list instead.

**Comment by Exxion** [comment]
The way they're implemented right now, no, it's not really fixable
It would be trivial if they used images instead of screen objects, but I'm... not really sure how client images work with clicking. Probably poorly.

**Comment by troots** [comment]
make them server side items spawned around the object and parented to it and actually tangible so that a clown can steal your buttons

**Comment by DeityLink** [comment]
They already are. if a clown runs away with the beaker, the buttons will run away with them.

**Comment by troots** [comment]
yeah but it's not the same as literally just stealing a button and rendering the ui broken, i remember a glitch that caused wizards to drop their spell verbs and the implications of such concepts being tangible items you can yoink and put in a pocket amuses me

**Comment by I-VAPE-VOX-CLOACA-EVERY-DAY-OF-MY-LIFE** [comment]
@deitylink the buttons won't move, that's the problem
